canu/inventory/plays/aruba-aoscx.yml

```diff
@@ -0,0 +1,140 @@
+---
+- hosts: sw-leaf-bmc-001
+  collections:
+    - arubanetworks.aoscx
+  gather_facts: true
+  vars:
+    # This should be a template
+    leaf_config: files/sw-leaf-bmc-001.cfg
```

Comment:
Can we make this path a variable?  Currently it's looking in `canu/canu/inventory/plays/files/sw-leaf-bmc-001.cfg` for switch configs.  Can we have a CLI argument for `--folder` ?

Reply:
It can be, I see now I just left in the config I was testing with hardcoded. 

An env var works better since the ansible-playbook can only accept `--list` and `--host`.

Reply:
Actually, we have some jinja templates in the repo already--can we just point it there?

Reply:
I made this a var for now.  The next iteration this should probably just be a callable role..